### DISCUSSION TITLE
`import { TSESLint } from 'typescript-eslint'`

### DISCUSSION THREAD

**Discussion by mightyiam** [typescript-eslint/typescript-eslint#9360] [Technical Discussions]
### Body

In order to avoid having both `typescript-eslint` and `@typescript-eslint/utils` as direct dependencies (devDeps, in our case), would it be reasonable to request that `TSESLint` would be exposed by `typescript-eslint`, please?

### Additional Info

Any additional info...

### Before you submit your discussion, please confirm the following. If any of these required steps are not taken, we may not be able to review your RFC. Help us to help you!

- [X] I have [read the discussions guidelines](https://typescript-eslint.io/contributing/discussions) and per those guidelines, this fits a discussion category, _not_ a help request or standard issue.
- [X] I have [searched for related discussions](https://github.com/typescript-eslint/typescript-eslint/discussions) and [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues) and found none that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

**Comment by bradzacher** [comment]
> In order to avoid having both typescript-eslint and @typescript-eslint/utils as direct dependencies

I'm not sure I understand - what's the issue with declaring your dependency here? Is it just that you don't want to declare two deps?

**Reply by mightyiam** [reply]
Exactly, yes. Just that.

**Reply by mightyiam** [reply]
Having two packages means there's a risk of their versions getting out-of-sync to mitigate.
